### PR TITLE
install monitoring service with provision.

### PR DESCRIPTION
**Pull request by shawn-hurley**
Adding playbook to install monitor service by passing a host a monitor RPM

### CODE REVIEW

File: rhui_provision/install_monitor_service.yml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+- name: Install Packages for Monitoring
+  hosts: MON_SERV
```
Comment (djwhatle) [inline rhui_provision/install_monitor_service.yml]: I tend to prefer fully written out variable names in Ansible playbooks when it makes sense. In this case I'm wondering whether MON_SERV means "Monitoring Service" or "Monitoring Server".

It's a nitpick, but I think it helps code readability in the long run.

File: rhui_provision/rhui_cloud_formation.template [hunk 2]
```diff
@@ -202,6 +202,67 @@
...
+                "ImageId": {
+                    "Fn::FindInMap": [
+                        "RHEL64", 
+                        {
+                            "Ref": "AWS::Region"
+                        }, 
+                        "AMI"
+                    ]
+                },
+                "BlockDeviceMappings": [
```
Comment (djwhatle) [inline rhui_provision/rhui_cloud_formation.template]: I don't think we need all of these disks for a monitoring instance. We set up separate partitions for /var/log, /var/lib/mongodb, /var/lib/pulp on RHUI instances.

File: rhui_provision/tasks/mon_serv_install.yml [hunk 3]
```diff
@@ -0,0 +1,12 @@
+- name: update machine to latest
```
Comment (djwhatle) [inline rhui_provision/tasks/mon_serv_install.yml]: I think we should capitalize the first letter in task names (helps them stand out when reading Ansible output, IMO), and also fully express what a task is doing. 

In this case, I would name this task "Yum update packages to latest available"

File: rhui_provision/rhui_cloud_formation.template (hunk 2 above)
Comment (shawn-hurley) [inline rhui_provision/rhui_cloud_formation.template]: Will update in the production one in cloude and here.

File: rhui_provision/tasks/mon_serv_install.yml (hunk 3 above)
Comment (shawn-hurley) [inline rhui_provision/tasks/mon_serv_install.yml]: +1

**Comment by djwhatle** [comment]
ACK

**Pull request merged**